examples/azure/terraform-azure-keyvault-example/README.md

```diff
@@ -0,0 +1,44 @@
+# Terraform Azure Availability Set Example
+
+This folder contains a simple Terraform module that deploys resources in [Azure](https://azure.microsoft.com/) to demonstrate
+how you can use Terratest to write automated tests for your Azure Terraform code. This module deploys a Key Vault with one secret, key, and certificate each.
+
+- A [Key Vault](https://azure.microsoft.com/en-us/services/key-vault/) that gives the module the following:
+    - [Secret](https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates)  with the value specified in the `secret_name`  output variable.
+    - [Key](https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates)  with the value specified in the `key_name`  output variable.
+    - [Certificate](https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates)  with the value specified in the `certificate_name`  output variable.
```

Comment:
I just noticed that in many of the links we have /en-us/ we should remove that so when the user navigates it picks up the culture setting in the browser.  Please do it for this PR, and we need a scrub of rest of contributions in a seperate PR.

Reply:
removed culture